spec/lib/bgs/form674_spec.rb

```diff
@@ -9,9 +9,13 @@
   let(:user_struct) { FactoryBot.build(:user_struct) }
   let(:saved_claim) { create(:dependency_claim_no_vet_information) }
 
+  before do
+    allow(Flipper).to receive(:enabled?).and_call_original
+  end
+
   context 'The flipper is turned on' do
     before do
-      Flipper.enable(:dependents_enqueue_with_user_struct)
+      allow(Flipper).to receive(:enabled?).with(:dependents_enqueue_with_user_struct).and_return(true)
```

Comment:
Change these to stubs instead of actually enabling/disabling the toggle. Added stub at the top to default to original values to handle any that are not stubbed.